src/ui/atoms/textField/TextField.tsx

```diff
@@ -3,6 +3,7 @@ import classNames from 'classnames'
 import type { RefObject } from 'react'
 import React from 'react'
 import { Typography } from '../typography/Typography'
+import { InputBase } from '../inputBase/InputBase'
 import './textField.scss'
 
 export type TextFieldProps = Readonly<{
```

Comment:
Maybe we could extend here the type from the `InputBaseProps`, but I'm not sure that StoryBook will correctly retrieve them in the arguments table, let's try it!

Reply:
Thanks for your feedback, storybook handles it very well!